pepper-apis/config/sqlproxy.env.ctmpl

```diff
@@ -1,16 +1,17 @@
-{{with $environment := env "ENVIRONMENT"}}
+{{$environment := env "ENVIRONMENT"}}
+{{$version := env "VERSION"}}
+{{$response := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+{{$conf := $response.Data}}
+
 {{if eq $environment "dev"}}
-       CLOUDSQL_CONNECTION_LIST=broad-ddp-dev:us-central1:pepper-dev=tcp:0.0.0.0:3307,broad-ddp-dev:us-central1:housekeeping-dev1=tcp:0.0.0.0:3308
+    CLOUDSQL_CONNECTION_LIST=broad-ddp-dev:us-central1:{{$conf.dbInstanceId}}=tcp:0.0.0.0:3307
```

Comment:
I think we can substitute broad-ddp-xxxx from {{$conf.googleProjectId}}
This way we can get rid of All if statements and just have below line in the template 
CLOUDSQL_CONNECTION_LIST={{$conf.googleProjectId}}:us-central1:{{$conf.dbInstanceId}}=tcp:0.0.0.0:3307